locales/en/case-studies.json

```diff
@@ -650,6 +650,15 @@
           "to": "https://protectai.com/blog/how-protect-ai-is-shaping-the-future-of-llm-security-with-ebpf"
         }
       ]
-    }
-  ]
-}
+   [
```

Comment:
This cannot be valid JSON. Did you try to validate the file, or better yet, run the website locally with the Docker instructions from the README?